packages/lesswrong/server/resolvers/postResolvers.ts

```diff
@@ -206,6 +227,22 @@ augmentFieldsDict(Posts, {
       }
     },
   },
+  dialogueMessageContents: {
+    resolveAs: {
+      type: 'String',
+      arguments: 'dialogueMessageId: String',
+      resolver: async (post: DbPost, args: {dialogueMessageId?: string}, context: ResolverContext): Promise<string|null> => {
+        const { currentUser } = context
+        const { dialogueMessageId } = args
+        if (!post.collabEditorDialogue) return null;
+        if (!dialogueMessageId) return null;
+        const isParticipant = currentUser ? isDialogueParticipant(currentUser?._id, post) : false
+        if (!isParticipant) return null;
```

Comment:
I think this would probably be clearer to read as:
```
if (!currentUser) return null;

const isParticipant = isDialogueParticipant(currentUser._id, post);
if (!isParticipant) return null;

return ...
```